<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>-30</y>
     <width>1291</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.21, y1:0.187773, x2:0.466, y2:1, stop:0 rgba(111, 127, 252, 255), stop:1 rgba(245, 221, 221, 255));</string>
   </property>
   <widget class="QComboBox" name="theme_box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 63, 124);
font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(155, 164, 247);</string>
    </property>
    <item>
     <property name="text">
      <string>---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Farmer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waitress</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chef</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Doctor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Programmer</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>1261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 164, 247);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	font: 32pt &quot;Rockwell&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.205, y1:0, x2:0.312, y2:1, stop:0 rgba(100, 241, 255, 0), stop:1 rgba(245, 221, 221, 0));</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonReady">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 24pt &quot;Rockwell&quot;;
</string>
    </property>
    <property name="text">
     <string>Ready?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 24pt &quot;Rockwell&quot;;</string>
    </property>
    <property name="text">
     <string> Avaible jobs:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>21</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 164, 247);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="greeting_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>891</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	font: 32pt &quot;Rockwell&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.205, y1:0, x2:0.312, y2:1, stop:0 rgba(100, 241, 255, 0), stop:1 rgba(245, 221, 221, 0));</string>
    </property>
    <property name="text">
     <string>Hello, user!</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>961</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 164, 247);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.205, y1:0, x2:0.312, y2:1, stop:0 rgba(100, 241, 255, 0), stop:1 rgba(245, 221, 221, 0));</string>
    </property>
    <property name="text">
     <string>Your lately previous
scores:</string>
    </property>
   </widget>
   <widget class="QLabel" name="prevlabel1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 20pt &quot;Rockwell&quot;;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> ---------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="prevlabel2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 20pt &quot;Rockwell&quot;;

border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> ---------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="prevlabel3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 20pt &quot;Rockwell&quot;;

border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> ---------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestlabel2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>480</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 20pt &quot;Rockwell&quot;;

border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> ---------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestlabel3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>560</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 20pt &quot;Rockwell&quot;;

border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> ---------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestlabel1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 20pt &quot;Rockwell&quot;;

border-radius: 10px;</string>
    </property>
    <property name="text">
     <string> ---------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>300</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.205, y1:0, x2:0.312, y2:1, stop:0 rgba(100, 241, 255, 0), stop:1 rgba(245, 221, 221, 0));</string>
    </property>
    <property name="text">
     <string>Your best overall 
scores:</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>20</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(155, 164, 247);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>750</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 98, 194);
color: rgb(255, 255, 255);
	font: 24pt &quot;Rockwell&quot;;</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>650</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="warning_label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.205, y1:0, x2:0.312, y2:1, stop:0 rgba(100, 241, 255, 0), stop:1 rgba(245, 221, 221, 0));</string>
    </property>
    <property name="text">
     <string>Please choose a profession!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>690</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.205, y1:0, x2:0.312, y2:1, stop:0 rgba(100, 241, 255, 0), stop:1 rgba(245, 221, 221, 0));</string>
    </property>
    <property name="text">
     <string>Hello there ^^</string>
    </property>
   </widget>
   <zorder>line_4</zorder>
   <zorder>theme_box</zorder>
   <zorder>label</zorder>
   <zorder>ButtonReady</zorder>
   <zorder>label_2</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>greeting_label</zorder>
   <zorder>line_3</zorder>
   <zorder>label_4</zorder>
   <zorder>prevlabel1</zorder>
   <zorder>prevlabel2</zorder>
   <zorder>prevlabel3</zorder>
   <zorder>bestlabel2</zorder>
   <zorder>bestlabel3</zorder>
   <zorder>bestlabel1</zorder>
   <zorder>label_11</zorder>
   <zorder>refresh_btn</zorder>
   <zorder>listView</zorder>
   <zorder>warning_label</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
